src/Factory/UrlAlternativeFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SitemapPlugin\Factory;
+
+use SitemapPlugin\Model\UrlAlternative;
+use SitemapPlugin\Model\UrlAlternativeInterface;
+
+final class UrlAlternativeFactory implements UrlAlternativeFactoryInterface
```

Comment:
What about `AlternativeUrl`? I know now it lists together in a file listing, but it sounds odd.